clio-client/src/main/scala/org/broadinstitute/clio/client/commands/MoveWgsUbamCommand.scala

```diff
@@ -31,63 +28,96 @@ object MoveWgsUbamCommand
     config: BaseArgs,
     ioUtil: IoUtil
   )(implicit ec: ExecutionContext): Future[HttpResponse] = {
-    config.location.foreach {
-      case "GCP" => ()
-      case _ =>
-        throw new Exception("Only GCP unmapped bams are supported at this time")
-    }
-    config.ubamPath.foreach {
-      case loc if loc.startsWith("gs://") => ()
-      case _ =>
-        throw new Exception("The destination of the ubam must be a cloud path")
-    }
     for {
-      wgsUbam <- queryForWgsUbam(webClient, config) withErrorMsg
-        """Could not query the WgsUbam. There may have been the incorrect number of WgsUbams returned.
-          |No files have been moved""".stripMargin
-      _ <- copyGoogleObject(wgsUbam.ubamPath, config.ubamPath, ioUtil) withErrorMsg
-        "An error occurred while moving the files in google cloud. No files have been moved"
+      _ <- verifyPath(webClient, config) withErrorMsg
+        "Clio client can only handle cloud operations right now."
+      wgsUbamPath <- queryForWgsUbamPath(webClient, config) withErrorMsg
+        "Could not query the WgsUbam. No files have been moved."
+      _ <- copyGoogleObject(wgsUbamPath, config.ubamPath, ioUtil) withErrorMsg
+        "An error occurred while copying the files in google cloud. No files have been moved."
       upsertUbam <- upsertUpdatedWgsUbam(webClient, config) withErrorMsg
         """An error occurred while upserting the WgsUbam. The state of clio is now inconsistent.
           |The ubam exists in both at both the old and the new locations.
           |At this time, Clio only knows about the bam at the old location.
           |Then, try upserting (not moving) the unmapped bam with the correct path then deleting the old bam.
```

Comment:
Is this a valid recovery scenario? E.g. is there a CLP that can update the metadata in this way?

If this kind of failure can occur, we should describe somewhere (confluence, etc) what the recovery process is. If we can delete the copied resource, then the recovery scenario is to run the same command again, once the underlying problem has been fixed.